recce/server.py

```diff
@@ -109,9 +104,18 @@ async def health_check(request: Request):
     return {"status": "ok"}
 
 
-class QueryInput(BaseModel):
-    base: Optional[bool] = False
-    sql_template: str
+@app.get("/api/info")
+async def get_info():
+    dbt_context = default_dbt_context()
+    try:
+        return {
+            'lineage': {
+                'base': dbt_context.get_lineage(base=True),
+                'current': dbt_context.get_lineage(base=False),
+            },
+        }
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=str(e))
 
 
 @app.get("/api/lineage")
```

Comment:
Seem we replace the `/api/lineage` by `/api/info`.
Can just remove the `GET /api/lineage` from our code base.